locales/es.js

```diff
@@ -0,0 +1,155 @@
+export const es = {
+  welcome: {
+    title: 'Abra su nueva cuenta bancaria',
+    description_p_1: 'Para abrir una cuenta bancaria, necesitaremos verificar su identidad.',
+    description_p_2: 'Solo tomará un par de minutos.',
+    next_button: 'Verificar identidad',
+  },
+  document_selector: {
+    title: 'Verifique su identidad',
+    hint: 'Seleccione el tipo de documento que desea cargar',
```

Comment:
I'd mostly heard 'subir' as the verb for uploading. Not sure which is more common though.